maths/perfect_number.py

```diff
@@ -70,7 +80,7 @@ def perfect(number: int) -> bool:
     try:
         number = int(input("Enter a positive integer: ").strip())
     except ValueError:
-        msg = "number must an integer"
+        msg = "number must be an integer"
         print(msg)
         raise ValueError(msg)
```

Comment:
```suggestion
        msg = "number must be an integer"
        raise ValueError(msg)
```
Nothing wrong with your changes, but the original code prints the error message before raising an error, and outputting the message twice is unnecessary.